contracts/template/Cargo.toml

```diff
@@ -0,0 +1,56 @@
+[package]
+name = "template"
+version = "1.0.0"
```

Comment:
let's set this to the same version as the other contracts. at some point i want to fix the crate hierarchy so everything will inherit